docs/modules/prometheus_ingester.md

```diff
@@ -15,6 +15,9 @@ For its usage example see [the prometheus example](/examples/prometheus).
 ```yaml
 # URL of the API for Prometheus queries such as https://foo.bar:9090
 apiUrl: <URL>
+# HTTP headers to send to API server
```

Comment:
```suggestion
# HTTP headers to send to the API server (in case of conflicting header names, the later has precedence)
```
Also, would you mind adding a test case for this situation? 

Such As
```yaml
httpHeaders:
  - name: foo
    value: bar
  - name: foo
    value: baz
```